src/main/java/io/strimzi/kafka/quotas/StaticQuotaCallback.java

```diff
@@ -234,15 +250,15 @@ public static MetricName metricName(String name, String type, String group) {
      * @return the MetricName object derived from the arguments.
      */
     public static MetricName metricName(String name, String type, String group, LinkedHashMap<String, String> tags) {
-        final String tagValues = tags.entrySet().stream().map(entry -> String.format("%s=%s", sanitise(entry.getKey()), sanitise(entry.getValue()))).collect(Collectors.joining(","));
+        final String tagValues = tags.entrySet().stream().map(entry -> java.lang.String.format("%s=%s", sanitise(entry.getKey()), sanitise(entry.getValue()))).collect(Collectors.joining(","));
         String mBeanName;
         final String sanitisedGroup = sanitise(group);
         final String sanitisedType = sanitise(type);
         final String sanitisedName = sanitise(name);
         if (!tagValues.isBlank()) {
-            mBeanName = String.format("%s:type=%s,name=%s,%s", sanitisedGroup, sanitisedType, sanitisedName, tagValues);
+            mBeanName = java.lang.String.format("%s:type=%s,name=%s,%s", sanitisedGroup, sanitisedType, sanitisedName, tagValues);
```

Comment:
Do we need to fullly qualify `String`?

Reply:
Resolved in 82bf79507c7c1e76c40808e722e04847cb265ec6